### ISSUE TITLE
Feature request: Show album cover rather than MusicBee logo

### ISSUE THREAD

**Issue by Newbytee** [Harmon758/mb_DiscordRichPresence#6]
As the title suggest, I would appreciate if on Discord the album cover was shown rather than the MusicBee logo, and perhaps the album name on hover.

**Comment by MichaelGatesDev** [comment]
As per the FAQ:
```
Q: Can I send images via the payload rather than uploading them to my Developer Dashboard?

Unfortunately, the SDK does not support this feature right now. However, we hear your desires! We know that a lot of games, like customization-heavy RPGs, would benefit greatly from being able to programmatically upload assets. It may be something we tackle in the future.
```
https://discordapp.com/developers/docs/rich-presence/faq

tl;dr it's not possible yet

**Comment by carks32** [comment]
though, @MichaelGatesDev, is it possible to make it load an image from somewhere else based on the Album tag? Like from last.fm, for example.

**Comment by MichaelGatesDev** [comment]
@carks32 Unfortunately it is not possible to stream any kind of image content to the RPC at this time. The best we could do is ask users to create their own application and use their own app id, and then write a script which auto uploads and removes album covers.. but it'd be clunky and unreliable,and there's no API for that part of Discord that I've found.

tl;dr no :(

**Comment by WSADKeysGaming** [comment]
I believe this is more possible now, as I see League Of Legends shows your character that you are playing as instead of the LoL logo.

**Comment by Newbytee** [comment]
@WSADKeysGaming I don't think that means anything. As earlier stated, you can upload images to the Developer Dashboard that Discord has and then display them, which is probably what League of Legends does. Doing this for every album cover in existence, or just popular ones, I think is unreasonable.

**Comment by scottzach1** [comment]
I hope I am not of any hindrance, however Spotify I believe has managed to achieve this.
Perhaps there have been changes since to allow this?

![image](https://user-images.githubusercontent.com/38968222/64933485-ae74d900-d899-11e9-89b3-b9a4bbaf1647.png)

**Comment by MichaelGatesDev** [comment]
> I hope I am not of any hindrance, however Spotify I believe has managed to achieve this.
> Perhaps there have been changes since to allow this?
> 
> ![image](https://user-images.githubusercontent.com/38968222/64933485-ae74d900-d899-11e9-89b3-b9a4bbaf1647.png)

Spotify partnered with Discord and as a result they were given some kind of special RP API access. It is still not possible.

**Comment by ghost** [comment]
Hello, sorry, newbie here. Just wanted to ask, is there a place where I can ask the Discord developers to make it so this is possible? Or vote on a preexisting feature request for Discord that mentions this? Sorry, this might be a dumb question lol. Thanks!

**Comment by ghost** [comment]
So I found a [repo](https://github.com/jojo2357/Music-Discord-Rich-Presence) where you can add album art to your own app in the Discord Developer Portal. This is how it looks like: 
![image](https://user-images.githubusercontent.com/66522068/111091107-e0d5c400-856c-11eb-929f-7c13f018a2f7.png)
![image](https://user-images.githubusercontent.com/66522068/111091155-01058300-856d-11eb-8471-e6839dd2b938.png)
![image](https://user-images.githubusercontent.com/66522068/111091162-05ca3700-856d-11eb-895b-616ac1a36d40.png)
Would it be possible to implement this here as well? I know that you can only have 150 album arts and that it may take some work but I still would find this pretty cool to be added. Do I need to create a new thread? (Also note: the linked repo is for Groove Music)
Update: The developer of the repo I linked added support for MusicBee, take a look [here](https://github.com/jojo2357/Music-Discord-Rich-Presence/discussions/20) if you're interested (please tell me if this is the wrong place to tell this stuff lmao).
